src/components/SecondaryNav/SecondaryNav.spec.js

```diff
@@ -1,77 +1,60 @@
 import React from 'react';
-import {shallow} from 'component-test-utils-react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import {SecondaryNav} from './index';
 
 describe('SecondaryNav', () => {
     it('should display children content', () => {
-        const wrapper = shallow(
-            <SecondaryNav header="">
-                content here
-            </SecondaryNav>
-        );
-
-        expect(wrapper.html()).toContain('content here');
+        render(<SecondaryNav header="">content here</SecondaryNav>);
+        expect(screen.queryByText('content here')).toBeInTheDocument();
     });
     it('should display a string in the header', () => {
-        const wrapper = shallow(<SecondaryNav header="my header">hello</SecondaryNav>);
-
-        expect(wrapper.html()).toContain('my header');
+        render(<SecondaryNav header="my header">content here</SecondaryNav>);
+        expect(screen.queryByText('my header')).toBeInTheDocument();
     });
 
     it('should add extra attribute', () => {
-        const wrapper = shallow(<SecondaryNav data-custom="test" header="my header">hello</SecondaryNav>);
-
-        expect(wrapper.html()).toContain('data-custom="test"');
+        render(<SecondaryNav data-testid="secondary-nav" data-custom="extra">content here</SecondaryNav>);
+        expect(screen.getByTestId('secondary-nav')).toHaveAttribute('data-custom', 'extra');
     });
 
     it('should have .secondaryNav_hidden when the menu is hidden', () => {
-        const wrapper = shallow(<SecondaryNav header="my header" isDefaultVisible={false}>hello</SecondaryNav>);
-
-        expect(wrapper.html()).toContain('secondaryNav_hidden');
+        render(<SecondaryNav data-testid="secondary-nav" isDefaultVisible={false}>content here</SecondaryNav>);
+        expect(screen.getByTestId('secondary-nav')).toHaveClass('moonstone-secondaryNav_hidden');
     });
 
     it('should not have .secondaryNav_hidden when the menu is visible', () => {
-        const wrapper = shallow(<SecondaryNav header="my header">hello</SecondaryNav>);
-
-        expect(wrapper.html()).not.toContain('secondaryNav_hidden');
+        render(<SecondaryNav data-testid="secondary-nav">content here</SecondaryNav>);
+        expect(screen.getByTestId('secondary-nav')).not.toHaveClass('moonstone-secondaryNav_hidden');
```

Comment:
We use a lot of classes for these tests. It does not respect the testing-libary [guideline](https://testing-library.com/docs/guiding-principles).
I suggest adding an `aria-expanded` attribute to the component and `aria-controls` to the button that toggles visibility.
- We will improve the semantic
- We will improve the accessible
- CSS class can change, not aria attribute

Can you change the tests to avoid the presence of CSS classes and use aria-* attributes when it possible ?